src/test/profile.test.js

```diff
@@ -0,0 +1,21 @@
+import { render, cleanup } from '@testing-library/react';
+import { Provider } from 'react-redux';
+import store from '../redux/store';
+import MissionProfile from '../components/MissionProfile';
+
+afterEach(cleanup);
+
+describe('Testing profile components', () => {
+  test('Generating the snapshots for MissionProfile', () => {
+    const member = {
+      mission_name: 'Test Profile',
+      member: true,
+    };
+    const { container } = render(
+      <Provider store={store}>
+        <MissionProfile member={member} />
+      </Provider>,
+    );
+    expect(container).toMatchSnapshot();
+  });
```

Comment:
- [ ] Here also, For the test, I have seen you have done a `Snapshot` test. this is great and important but we need to write unit tests with React Testing Library and Jest (for unit tests) alongside the snapshot tests.

Reply:
Resolved